### PR TITLE
Add MAC and Windows information for installation

### PR DESCRIPTION
**Pull request by rhatdan**
We are getting a lot of users, confused about how to install and use
Podman on Windows and MAC machines, they are missing the links on
the libpod README.md files.

We need  this information on the getting started page as well.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @vrothberg @baude @jwhonce PTAL

**Review by vrothberg — COMMENTED** [review]
LGTM

File: getting-started/installation.md [hunk 1]
```diff
@@ -7,6 +7,34 @@ title: Podman Installation
...
+
+#### Remote Client
+  * [Latest remote client for MacOS](https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-darwin-amd64.zip)
+
+### Windows
+
+Podman is a tool for running Linux containers.  You can do this from a Windows
+desktop as long as you have access to a linux box either running inside of a
+VM on the host, or available via the network.  You need to install the remote
+client and then setup ssh connection informantion in the podman-remote.conf
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: informantion -> information

File: getting-started/installation.md [hunk 2]
```diff
@@ -7,6 +7,34 @@ title: Podman Installation
 
 ## Installing packaged versions of Podman
 
+### MacOS
+
+Podman is a tool for running Linux containers.  You can do this from a MacOS
+desktop as long as you have access to a linux box either running inside of a
+VM on the host, or available via the network.  You need to install the remote
+client and then setup ssh connection informantion in the podman-remote.conf
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: informantion -> information

File: getting-started/installation.md [hunk 3]
```diff
@@ -7,6 +7,34 @@ title: Podman Installation
...
+  * [Latest remote client for MacOS](https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-darwin-amd64.zip)
+
+### Windows
+
+Podman is a tool for running Linux containers.  You can do this from a Windows
+desktop as long as you have access to a linux box either running inside of a
+VM on the host, or available via the network.  You need to install the remote
+client and then setup ssh connection informantion in the podman-remote.conf
+file.  Podman can also be run in the Windows Subsystem for Linux system, check
+out the link below to describe how this is done.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md]: describe -> see a description of

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Pull request merged**